### DISCUSSION TITLE
Can I address 4 y motors at the same time with GRBLhal?

### DISCUSSION THREAD

**Discussion by Timacaro** [grblHAL/core#592] [Q&A]
God afternoon,

Got a 6 pack card and I want to address 4 y motors with GRBLhal. Can I do that?

**Comment by terjeio** [comment]
What do you mean by addressing 4 Y motors? Separately or simultaneously?
If simultaneously hacking the board might be the easiest option.

**Reply by Timacaro** [reply]
Actually yes, I want address 4 motors at the same time. Hacking you say. well that's not really my thing. What you do you mean by hacking, changing what?

**Reply by terjeio** [reply]
Depends on the drivers you use, if not UART or SPI controlled Trinamics I would cut PCB traces to step and dir signals to motors 4 and 5 (counting X as 0) and solder wires from step/dir pins of motor 1 to 4 (Y0 to Y2) and 3 to 5 (Y1 to Y3) and configure Y as ganged. If you use Trinamic drivers you are in for some major code changes instead.
BTW the simplest way to achive this is to use external drivers and just wire the Y drivers in paralell.
